### PR TITLE
Add link to CEL approver-policy plugin

### PR DESCRIPTION
**Pull request by erikgb**
After a nice meeting with some of the cert-manager maintainers in today's standup, I am suggesting adding a link to my experimental CEL approver-policy plugin.

### CODE REVIEW

File: content/docs/projects/approver-policy/README.md [hunk 1]
```diff
@@ -415,7 +415,9 @@ spec:
         val-1: key-1
 ```
 
-There are currently no open source plugins.
+There is currently only one (experimental) open source plugin:
```
Comment (JoshVanL) [inline content/docs/projects/approver-policy/README.md:418]:
```suggestion
## Known Plugins
```

Could you also add a note at the end of this section to invite authors to share their plugins back to this page?

**Review by JoshVanL — CHANGES_REQUESTED** [review]
Thanks for sharing @erikgb, this looks great :smile:

File: content/docs/projects/approver-policy/README.md (hunk 1 above)
Comment (erikgb) [reply content/docs/projects/approver-policy/README.md]: Better now?

**Comment by JoshVanL** [comment]
Thanks @erikgb!

/approve
/lgtm

**Pull request merged**